x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/create.ts

```diff
@@ -51,7 +53,14 @@ export const registerSiemRuleMigrationsCreateRoute = (
 
             const ctx = await context.resolve(['securitySolution']);
             const ruleMigrationsClient = ctx.securitySolution.getSiemRuleMigrationsClient();
-
+            const auditLogger = ctx.securitySolution.getAuditLogger();
+            if (auditLogger) {
```

Comment:
We separate the if condition with the .log method call because its more consistent between read and write operations (Read needs to be logged after success, while write needs to be logged before), and we cannot initialize the class later on reads because on errors the siemMigrationAuditLogger would be undefined.